src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureBaseForm.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { ReactNode, useCallback } from 'react';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+import { defineMessages } from 'react-intl';
+
+import { useLoggedInUser } from '~data/index';
+import { InputLabel, SelectOption, FormSection } from '~core/Fields';
+import UserAvatar from '~core/UserAvatar';
+import UserMention from '~core/UserMention';
+import ColorTag, { Color } from '~core/ColorTag';
+import DomainDropdown from '~core/DomainDropdown';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+
+import BalanceSelect from './BalanceSelect';
+import { tokens as tokensData } from './constants';
+import styles from './ExpenditureSettings.css';
+import { Props } from './ExpenditureSettings';
+
+export const MSG = defineMessages({
+  team: {
+    id:
+      'dashboard.ExpenditurePage.ExpenditureSettings.ExpenditureBaseForm.team',
+    defaultMessage: 'Team',
+  },
+  balance: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.ExpenditurePage.ExpenditureSettings.ExpenditureBaseForm.balance',
```

Comment:
Here you can put it in backticks `` and then you don't need the disable comment.